### PR TITLE
Refactor SDK list to table with docs column

### PR DESCRIPTION
**Pull request by vgvoleg**
### Changelog entry 

...

### Changelog category 

* Documentation (changelog entry is not required)

### Additional information

...

### CODE REVIEW

File: ydb/docs/en/core/reference/ydb-sdk/_includes/index.md [hunk 1]
```diff
@@ -4,16 +4,18 @@
...
-- Python [https://github.com/ydb-platform/ydb-python-sdk](https://github.com/ydb-platform/ydb-python-sdk)
-- Rust [https://github.com/ydb-platform/ydb-rs-sdk](https://github.com/ydb-platform/ydb-rs-sdk)
+| С# (.NET) | [https://github.com/ydb-platform/ydb-dotnet-sdk](https://github.com/ydb-platform/ydb-dotnet-sdk) | N/A |
+| Go | [https://github.com/ydb-platform/ydb-go-sdk/v3](https://github.com/ydb-platform/ydb-go-sdk) | [https://pkg.go.dev/github.com/ydb-platform/ydb-go-sdk/v3](https://pkg.go.dev/github.com/ydb-platform/ydb-go-sdk/v3) |
+| Java | [https://github.com/ydb-platform/ydb-java-sdk](https://github.com/ydb-platform/ydb-java-sdk) | N/A |
+| Node.js | [https://github.com/ydb-platform/ydb-nodejs-sdk](https://github.com/ydb-platform/ydb-nodejs-sdk) | N/A |
+| PHP | [https://github.com/ydb-platform/ydb-php-sdk](https://github.com/ydb-platform/ydb-php-sdk) | N/A |
+| Python | [https://github.com/ydb-platform/ydb-python-sdk](https://github.com/ydb-platform/ydb-python-sdk) | [https://ydb-platform.github.io/ydb-python-sdk/](https://ydb-platform.github.io/ydb-python-sdk/) |
+| Rust | [https://github.com/ydb-platform/ydb-rs-sdk](https://github.com/ydb-platform/ydb-rs-sdk) | N/A |
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/_includes/index.md:18]:
```suggestion
| Go | [github.com/ydb-platform/ydb-go-sdk/v3](https://github.com/ydb-platform/ydb-go-sdk) | [pkg.go.dev/github.com/ydb-platform/ydb-go-sdk/v3](https://pkg.go.dev/github.com/ydb-platform/ydb-go-sdk/v3) |
| Java | [github.com/ydb-platform/ydb-java-sdk](https://github.com/ydb-platform/ydb-java-sdk) | N/A |
| Node.js | [github.com/ydb-platform/ydb-nodejs-sdk](https://github.com/ydb-platform/ydb-nodejs-sdk) | N/A |
| PHP | [github.com/ydb-platform/ydb-php-sdk](https://github.com/ydb-platform/ydb-php-sdk) | N/A |
| Python | [github.com/ydb-platform/ydb-python-sdk](https://github.com/ydb-platform/ydb-python-sdk) | [ydb-platform.github.io/ydb-python-sdk](https://ydb-platform.github.io/ydb-python-sdk/) |
| Rust | [https://github.com/ydb-platform/ydb-rs-sdk](https://github.com/ydb-platform/ydb-rs-sdk) | N/A |
```

The table is a little too wide; this is one possible way to make it a bit narrower.

Comment (blinkov) [reply ydb/docs/en/core/reference/ydb-sdk/_includes/index.md:18]: The same for the ru version

**Review by blinkov — DISMISSED** [review]

File: ydb/docs/en/core/reference/ydb-sdk/_includes/index.md [hunk 2]
```diff
@@ -4,16 +4,18 @@
...
 {% endif %}
-- C# (.NET) [https://github.com/ydb-platform/ydb-dotnet-sdk](https://github.com/ydb-platform/ydb-dotnet-sdk)
-- Go [https://github.com/ydb-platform/ydb-go-sdk/v3](https://github.com/ydb-platform/ydb-go-sdk)
-- Java [https://github.com/ydb-platform/ydb-java-sdk](https://github.com/ydb-platform/ydb-java-sdk)
-- Node.js [https://github.com/ydb-platform/ydb-nodejs-sdk](https://github.com/ydb-platform/ydb-nodejs-sdk)
-- PHP [https://github.com/ydb-platform/ydb-php-sdk](https://github.com/ydb-platform/ydb-php-sdk)
-- Python [https://github.com/ydb-platform/ydb-python-sdk](https://github.com/ydb-platform/ydb-python-sdk)
-- Rust [https://github.com/ydb-platform/ydb-rs-sdk](https://github.com/ydb-platform/ydb-rs-sdk)
+| С# (.NET) | [ydb-platform/ydb-dotnet-sdk](https://github.com/ydb-platform/ydb-dotnet-sdk) | N/A |
+| Go | [ydb-platform/ydb-go-sdk](https://github.com/ydb-platform/ydb-go-sdk) | [pkg.go.dev/github.com/ydb-platform/ydb-go-sdk/v3](https://pkg.go.dev/github.com/ydb-platform/ydb-go-sdk/v3) |
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/_includes/index.md:13]:
```suggestion
| Go | [ydb-platform/ydb-go-sdk](https://github.com/ydb-platform/ydb-go-sdk) | [https://pkg.go.dev/github.com/ydb-platform/ydb-go-sdk/v3](https://pkg.go.dev/github.com/ydb-platform/ydb-go-sdk/v3) |
```

File: ydb/docs/en/core/reference/ydb-sdk/_includes/index.md [hunk 3]
```diff
@@ -4,16 +4,18 @@
...
+| Java | [ydb-platform/ydb-java-sdk](https://github.com/ydb-platform/ydb-java-sdk) | N/A |
+| Node.js | [ydb-platform/ydb-nodejs-sdk](https://github.com/ydb-platform/ydb-nodejs-sdk) | N/A |
+| PHP | [ydb-platform/ydb-php-sdk](https://github.com/ydb-platform/ydb-php-sdk) | N/A |
+| Python | [ydb-platform/ydb-python-sdk](https://github.com/ydb-platform/ydb-python-sdk) | [ydb-platform.github.io/ydb-python-sdk](https://ydb-platform.github.io/ydb-python-sdk/) |
```
Comment (blinkov) [inline ydb/docs/en/core/reference/ydb-sdk/_includes/index.md:17]:
```suggestion
| Python | [ydb-platform/ydb-python-sdk](https://github.com/ydb-platform/ydb-python-sdk) | [https://ydb-platform.github.io/ydb-python-sdk](https://ydb-platform.github.io/ydb-python-sdk/) |
```

File: ydb/docs/ru/core/reference/ydb-sdk/index.md [hunk 4]
```diff
@@ -4,18 +4,18 @@
...
 {% endif %}
-- С# (.NET) [https://github.com/ydb-platform/ydb-dotnet-sdk](https://github.com/ydb-platform/ydb-dotnet-sdk)
-- Go [https://github.com/ydb-platform/ydb-go-sdk/v3](https://github.com/ydb-platform/ydb-go-sdk)
-- Java [https://github.com/ydb-platform/ydb-java-sdk](https://github.com/ydb-platform/ydb-java-sdk)
-- Node.js [https://github.com/ydb-platform/ydb-nodejs-sdk](https://github.com/ydb-platform/ydb-nodejs-sdk)
-- PHP [https://github.com/ydb-platform/ydb-php-sdk](https://github.com/ydb-platform/ydb-php-sdk)
-- Python [https://github.com/ydb-platform/ydb-python-sdk](https://github.com/ydb-platform/ydb-python-sdk)
-- Rust [https://github.com/ydb-platform/ydb-rs-sdk](https://github.com/ydb-platform/ydb-rs-sdk)
+| С# (.NET) | [ydb-platform/ydb-dotnet-sdk](https://github.com/ydb-platform/ydb-dotnet-sdk) | N/A |
+| Go | [ydb-platform/ydb-go-sdk](https://github.com/ydb-platform/ydb-go-sdk) | [pkg.go.dev/github.com/ydb-platform/ydb-go-sdk/v3](https://pkg.go.dev/github.com/ydb-platform/ydb-go-sdk/v3) |
```
Comment (blinkov) [inline ydb/docs/ru/core/reference/ydb-sdk/index.md:13]:
```suggestion
| Go | [ydb-platform/ydb-go-sdk](https://github.com/ydb-platform/ydb-go-sdk) | [https://pkg.go.dev/github.com/ydb-platform/ydb-go-sdk/v3](https://pkg.go.dev/github.com/ydb-platform/ydb-go-sdk/v3) |
```

File: ydb/docs/ru/core/reference/ydb-sdk/index.md [hunk 5]
```diff
@@ -4,18 +4,18 @@
...
+| Java | [ydb-platform/ydb-java-sdk](https://github.com/ydb-platform/ydb-java-sdk) | N/A |
+| Node.js | [ydb-platform/ydb-nodejs-sdk](https://github.com/ydb-platform/ydb-nodejs-sdk) | N/A |
+| PHP | [ydb-platform/ydb-php-sdk](https://github.com/ydb-platform/ydb-php-sdk) | N/A |
+| Python | [ydb-platform/ydb-python-sdk](https://github.com/ydb-platform/ydb-python-sdk) | [ydb-platform.github.io/ydb-python-sdk](https://ydb-platform.github.io/ydb-python-sdk/) |
```
Comment (blinkov) [inline ydb/docs/ru/core/reference/ydb-sdk/index.md:17]:
```suggestion
| Python | [ydb-platform/ydb-python-sdk](https://github.com/ydb-platform/ydb-python-sdk) | [https://ydb-platform.github.io/ydb-python-sdk](https://ydb-platform.github.io/ydb-python-sdk/) |
```

**Review by fomichev3000 — APPROVED** [review]

**Pull request merged**